models/spring-ai-azure-openai/src/main/java/org/springframework/ai/azure/openai/AzureOpenAiChatClient.java

```diff
@@ -137,6 +181,44 @@ public ChatResponse generate(Prompt prompt) {
 			.withPromptMetadata(generatePromptMetadata(chatCompletions));
 	}
 
+	@Override
+	public Flux<ChatResponse> generateStream(Prompt prompt) {
+
+		ChatCompletionsOptions options = toAzureChatCompletionsOptions(prompt);
+		options.setStream(true);
+
+		IterableStream<ChatCompletions> chatCompletionsStream = this.openAIClient
+			.getChatCompletionsStream(this.getModel(), options);
+
+		return Flux.fromStream(chatCompletionsStream.stream()
+			// Note: the first chat completions can be ignored when using Azure OpenAI
+			// service which is a
+			// known service bug.
+			.skip(1)
+			.map(ChatCompletions::getChoices)
+			.flatMap(List::stream)
+			.map(choice -> {
+				System.out.println(choice.getDelta());
```

Comment:
Leaving the System.out.println as just an accidental leftover? 🤔

Reply:
right, good catch!